modules/persistent-storage-local-install.adoc

```diff
@@ -98,7 +89,7 @@ metadata:
   name: local-storage-operator
   namespace: openshift-local-storage
 spec:
-  channel: "${OC_VERSION}"
+  channel: stable
```

Comment:
```suggestion
  channel: stable <2>
```

Reply:
The companion 2nd item would go below the <1> item and read something like:
<2> The stable channel is required for access to the needed operator.

Reply:
Please feel free to edit or change my wording on a technical basis. I'm only alerting you to point out that the new "stable" channel might need to be identified so the customer is more likely to see it.

Reply:
Thanks.